cpu/native/periph/timer.c

```diff
@@ -155,30 +153,6 @@ int timer_clear(tim_t dev, int channel)
     return 1;
 }
 
-void timer_irq_enable(tim_t dev)
-{
-    (void)dev;
-    DEBUG("%s\n", __func__);
-
-    if (register_interrupt(SIGALRM, native_isr_timer) != 0) {
```

Comment:
Just realized that this is the only point in native where this callback is registered. Maybe someone with more know-how should do this ^^"